Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -386,7 +386,7 @@ def audit_deps
           if @new_formula &&
              dep_f.keg_only_reason&.reason == :provided_by_macos &&
              dep_f.keg_only_reason.valid? &&
-             !%w[apr apr-util openblas openssl openssl@1.1].include?(dep.name)
+             !%w[apr apr-util openblas openssl openssl@1.1 python@2].include?(dep.name)
```

Comment:
This is a [whitelist](https://github.com/Homebrew/brew/commit/7812f6984e8696b14d4e87c8aee0b7e36edce883), not a blacklist.

Reply:
yeah, this section is about whitelisting keg-only dependencies, and I don't think `python@2` is keg-only

perhaps it would be better to copy what is done to blacklist depending on `git`?

* https://github.com/Homebrew/brew/blob/ce83253864c89bfa456a7ab321698731ffbd7373/Library/Homebrew/dev-cmd/audit.rb#L410

Reply:
Might be also worth porting that to reference `uses_from_macos` and providing an opt-out for relevant formulae depending on Git